util/collections/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod garbage;
```

Comment:
It would be nice to add a crate-level comment explaining what it is about.
Like, what collections could we provide here, other than garbage collections? 😁

Reply:
Okay